fcrepo-transform/src/main/java/org/fcrepo/transform/TransformationFactory.java

```diff
@@ -31,18 +32,20 @@
  * Get a Transformation from a MediaType
  *
  * @author cbeer
+ * @author ajs6f
  */
 public class TransformationFactory {
 
-    private Map<String, Transformation<?>> mimeToTransform = new HashMap<>();
+    @SuppressWarnings("rawtypes")
+    private final Map<String, Function<InputStream, Transformation>> mimeToTransform = new HashMap<>();
 
     /**
      * Get a new TransformationFactory with the default classes
      * @throws SecurityException if security exception occurred
      */
     public TransformationFactory() {
-        mimeToTransform.put(contentTypeSPARQLQuery, new SparqlQueryTransform(null));
-        mimeToTransform.put(APPLICATION_RDF_LDPATH, new LDPathTransform(null));
+        mimeToTransform.put(contentTypeSPARQLQuery, is -> new SparqlQueryTransform(is));
```

Comment:
Same comment as in earlier PR:
`mimeToTransform.put(contentTypeSPARQLQuery, SparqlQueryTransform::new);`